content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -44,14 +44,11 @@ external IP address.
 
 1. Run a Hello World application in your cluster:
 
-{{< codenew file="service/load-balancer-example.yaml" >}}
+    {{< codenew file="service/load-balancer-example.yaml" >}}
 
-```shell
-kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
-```
+        kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
```

Comment:
Instead of removing the explicit code fences (i.e. the original line 49 and 51), please keep them and indent the three lines by 3 spaces.

Reply:
fixed